src/include/sof/schedule/ll_schedule.h

```diff
@@ -33,6 +33,12 @@
 
 #define ll_sch_get_pdata(task) task->private
 
+#if __XCC__
+#define TICKS_NEEDED_TO_FINISH_RESCHEDULE 80
```

Comment:
On what platform have you measured it? I think it will differ for different XTAL frequencies.

Reply:
The amount of ticks should be clock frequency independant. However it is DSP and compiler dependant so I related it with compiler used. I think it should be moved to arch as it is valid only for xtensa ISA